src/style.css

```
@@ -0,0 +1,104 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+  border: none;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: inherit;
+  font-size: 16px;
+  font-weight: 300;
+  /* font-weight: 300; */
+  box-sizing: border-box;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  /* font-weight: bold; */
+  outline: none;
+  font-weight: 800;
```

Do we need these comments?

```suggestion
  outline: none;
  font-weight: 800;
```